packages/fiori/src/Timeline.js

```diff
@@ -136,6 +137,31 @@ class Timeline extends UI5Element {
 			}
 		}
 	}
+
+	_onkeydown(event) {
+		if (isTabNext(event)) {
+			if (!event.target.nameClickable || event.isMarked === "link") {
+				this._handleTabNextOrPrevious(event, isTabNext(event));
+			}
+		} else if (isTabPrevious(event)) {
+			this._handleTabNextOrPrevious(event);
+		}
+	}
+
+	_handleTabNextOrPrevious(event, isNext) {
```

Comment:
IMO this method should not return anything.

Reply:
Done